package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "postcss-elm-tailwind",
   "main": "./index.js",
-  "version": "0.10.0",
+  "version": "0.10.1",
```

```suggestion
  "version": "0.10.0",
```

Changes to the version are handled by our release process already.